app/assets/stylesheets/_normalize.css.scss

```diff
@@ -0,0 +1,425 @@
+/*! normalize.css v3.0.1 | MIT License | git.io/normalize */
```

Comment:
since it is an external css lib it belongs in `vendor/assets/stylesheets/` folder